scripts/tasks.ts

```diff
@@ -56,3 +56,20 @@ export async function upgradeProxy(
   console.log(`Proxy ${contractName} deployed to:`, contract.address);
   console.log(`Impl ${contractName} deployed to:`, contractImplAddress);
 }
+
+export async function deployNonUpgradeableContract(
```

Comment:
What is the difference between this and `deployDirect`?

Reply:
ohh yes, both are same.
Anyway we don't require it.
Will remove.